web/crux/src/main.ts

```diff
@@ -77,6 +83,10 @@ const bootstrap = async () => {
   const apiOptions = loadGrpcOptions('api', configService.get<string>('GRPC_API_PORT'))
   const httpOptions = configService.get<string>('HTTP_API_PORT', '1848')
 
+  app.useGlobalGuards(app.get(JwtAuthGuard))
```

Comment:
If we are using globbally, we should delete from controllers.

Reply:
yeah that's the idea.